File: check_trans.py
t = open('lib/translations.ts', 'r', encoding='utf-8').read()
s = t[t.find('dahuiyulu: ['):]
e = s.find('\n  ],')
section = s[:e]
lines = [l for l in section.split('\n') if l.strip().startswith("'")]
print(f'实际条目数: {len(lines)}')
print(f'前3行: {lines[:3]}')
print(f'后3行: {lines[-3:]}')

File: _check_trans.js
const fs = require('fs');
const t = fs.readFileSync('lib/translations.ts', 'utf8');
const idx = t.indexOf('dahuiyulu:');
const end = t.indexOf('\n  ],', idx);
const section = t.substring(idx, end);
const lines = section.split('\n').filter(l => l.trim().startsWith("'"));
console.log('Current translation segments:', lines.length);
